### PR TITLE
[BleBox] implemented update firmware

### PR DESCRIPTION
**Pull request by szczukot**

### CODE REVIEW

File: www/app/HardwareController.js [hunk 1]
```diff
@@ -5276,7 +5312,7 @@ define(['app'], function (app) {
                 $("#hardwarecontent #divhttppoller").hide();
                 if (text.indexOf("Open Weather Map") >= 0)
                 {
-                    $("#hardwarecontent #hardwareparamsunderground #location").val("lat=53.40&lon=14.58");
+                    $("#hardwarecontent #hardwareparamsunderground #location").val(data["Password"]);
```
Comment (gizmocuz) [inline www/app/HardwareController.js:5315]: Is this change correct? not really blebox firmware updated related...

**Review by gizmocuz — COMMENTED** [review]
One question...

File: www/app/HardwareController.js (hunk 1 above)
Comment (szczukot) [reply www/app/HardwareController.js:5315]: This is occasion to test for https://www.domoticz.com/forum/viewtopic.php?p=116842#p116842

Comment (gizmocuz) [reply www/app/HardwareController.js:5315]: Okey thanks, maybe best to next time make two pull requests ;)

**Comment by gizmocuz** [comment]
Don't forget an version increase in the appcache file

**Comment by gizmocuz** [comment]
https://github.com/domoticz/domoticz/commit/a858742cc7a681e720b3b06a4b600e6e18b10b0e

**Pull request merged**